test/async-hooks/test-async-functions-context.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+const common = require('../common');
```

Comment:
To appease the linter:

```suggestion
require('../common');
```